source/connect.txt

```diff
@@ -9,5 +9,174 @@ Connect to a Deployment
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+.. important::
+
+   Ensure that a MongoDB deployment is running before attempting to 
```

Comment:
[nit] Maybe something like "Ensure that you have a running MongoDB deployment before attempting to connect with ``mognosh``" ?

Reason being, as written it sounds like "ensure that somewhere in the world a MongoDB deployment is running before you connect."